modules/sb-provision-template-application.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/provisioning-template-application-bundle.adoc
+
+[id='sb-provision-template-application-{context}']
+= Provisioning template applications
+
+The following procedure provisions an example PostgreSQL template application
+that was made available by the template service broker.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed and started the template service broker.
+
+.Procedure
+
+. Create a project.
+.. Navigate in the web console to *Home* -> *Projects* and click
+*Create Project*.
+.. Enter `test-postgresql` in the *Name* field and click *Create*.
+
+. Create a service instance.
+.. Navigate to the *Catalog* -> *Developer Catalog* page.
+.. Select the *PostgreSQL (Ephemeral)* template application and click
+*Create Service Instance*.
+.. Review the default selections and set any other required fields, and click
+*Create*.
+.. Go to *Catalog* -> *Provisioned Services* and verify that the
+*postgresql-ephemeral* service instance is created and has a status of *Ready*.
++
+You can check the progress on the *Home* -> *Events* page. After a few moments,
+you should see an event for *postgresql-ephemeral* with the message "The
+instance was provisioned successfully".
+
+. Create a service binding.
++
+.. From the *Provisioned Services* page, click *postgresql-ephemeral* and click
+*Create Service Binding*.
+.. Review the default service binding name and click *Create*.
++
+This creates a new secret for binding using the name provided.
+
+. Review the secret that was created.
+.. Navigate to *Workloads* -> *Secrets* and verify that a secret was created,
```

Comment:
Same comment applies here about this sentence feeling clunky. Possibly move the secret name to immediately after where it's mentioned:

> ...verify that a secret, named **postgresql-ephemeral**, was created.